lib/src/disk_usage/disk_usage_view.dart

```diff
@@ -48,23 +51,61 @@ class _DiskUsageViewState extends State<DiskUsageView> {
         label: const Text("Export Logs"),
         icon: const Icon(Icons.bug_report),
         onPressed: () async {
-          // i would use "saveFile" but for some reason it crashes
-          final dir = await FilePicker.platform.getDirectoryPath();
-          if (dir == null) return;
-          final path = p.join(dir, "ditto_log.txt");
-          await DittoLogger.exportLogs(path);
-          _showSnackbar("Logs exported to $path");
+          try {
+            String exportPath;
+            
+            if (Platform.isIOS) {
+              // On iOS, export to the app's Documents directory
+              final documentsDir = await getApplicationDocumentsDirectory();
+              exportPath = p.join(documentsDir.path, "ditto_log.txt");
+            } else {
+              // On Android, macOS, and other platforms, let user choose directory
+              final dir = await FilePicker.platform.getDirectoryPath();
```

Comment:
If we use `saveFile` here, it will work on web as well - the user will have to provide the name, but arguably this is more useful anyways if they are exporting multiple times. We can set a default filename using `saveFile` anyways